Frontend/ui-driver/src/Screens/EnterMobileNumberScreen/Controller.purs

```diff
@@ -71,12 +71,12 @@ eval (PrimaryEditTextAction (PrimaryEditText.TextChanged valId newVal)) state =
   _ <- if length newVal == 10 then do
             pure $ hideKeyboardOnNavigation true 
             else pure unit    
-  let isValidMobileNumber = case (charAt 0 newVal) of 
-                                    Just a -> if a=='0' || a=='1' || a=='2' || a=='5' then false 
-                                                else if a=='3' || a=='4' then
-                                                    if newVal=="4000400040" || newVal=="3000300030" then true else false 
-                                                        else true 
-                                    Nothing -> true 
+  let isValidMobileNumber = true --case (charAt 0 newVal) of 
```

Comment:
why is this removed? It's a mandatory item for getting approval on play store/ app store